setup.py

```diff
@@ -69,5 +70,6 @@
         'Programming Language :: Python :: 3.8',
         'Topic :: Scientific/Engineering :: Bio-Informatics',
     ],
+    use_scm_version={"local_scheme": "no-local-version"},
```

Comment:
weird, why/how was it working before?

Reply:
This PR brought back the way it worked before. During the PR of migration to v2, this line was lost.